src/azure-cli/azure/cli/command_modules/serviceconnector/_resource_config.py

```diff
@@ -183,6 +183,18 @@ class CLIENT_TYPE(Enum):
 }
 
 
+# The dict defines the required parameters used in the source resources for creation.
+SOURCE_RESOURCES_CREATE_PARAMS = {
```

Comment:
can we put it in `SOURCE_RESOURCES_PARAMS`? what's the impact? will it bring more complexity in interactive exp implementation?